recipe-box-frontend/src/main/frontend/src/components/RecipeView/RecipeView.tsx

```diff
@@ -0,0 +1,26 @@
+import Header from "../Header/Header";
+import Footer from "../Footer/Footer";
+import {useState} from "react";
+import Categories from "../RecipesList/Categories";
+import "./RecipeView.css";
+import RecipeViewGrid from "./RecipeViewGrid";
+
+
+export default function RecipeView() {
```

Comment:
If this is a page, better move RecipeView inside the Pages folder, while RecipeDetails and RecipeImageDetails remain as components